example.yaml

```diff
@@ -410,6 +410,18 @@ splunk_hec_ingest_timeout: "10ms"
 # indicator=true set, or if they have a trace ID of 0.
 splunk_span_sample_rate: 10
 
+# (optional) The maximum duration to keep an HEC submission HTTP
+# request. After this duration, veneur will close & re-open the HTTP
+# connection even if less than `splunk_hec_batch_size` have been
+# ingested. This defaults to the flush `interval` setting (10s).
+splunk_hec_max_connection_lifetime: "10s"
+
+# (optional) The maximum (random) time to jitter an HEC request
```

Comment:
I had a hard time understanding this at first read. From what I understand this is meant to "jitter" when connections are made, right? Maybe something like "The upper bound of a random 'jitter' added to each connection's allowed lifetime, causing workers to randomly distribute their reconnects across `max(this setting)` time to reduce connection storms."

This is just a suggestion, take it or leave it! <3

Reply:
good suggestion! Will fix!